src/main.cpp

```diff
@@ -21,6 +21,9 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
  *
 \******************************************************************************/
+#ifdef _MSC_VER
+#    pragma comment( linker, "/subsystem:windows /ENTRY:mainCRTStartup" )
+#endif
```

Comment:
Could you please add a comment to this what this does Linker Flags? Maybe move it elsewhere down if possible

Reply:
Marking resolved in favor of this thread: https://github.com/jamulussoftware/jamulus/pull/2299#discussion_r851670516